docs/standard/base-types/best-practices-strings.md

```diff
@@ -27,7 +27,7 @@ This article examines the string sorting, comparison, and casing methods in .NET
 
 ## Recommendations for string usage
 
-When you develop with .NET, follow these recommendations when you compare strings:
+When you develop with .NET, follow these recommendations when you compare strings (for example, when you use <xref:System.String.Equals%2A?displayProperty=nameWithType>, <xref:System.String.Compare%2A?displayProperty=nameWithType>, <xref:System.String.IndexOf%2A?displayProperty=nameWithType> <xref:System.String.StartsWith%2A?displayProperty=nameWithType>, or related methods):
```

Comment:
Unless you're already familiar with the terminology, it's quite hard to know that "string comparison" refers to more than just `.Equals` and `.CompareTo`. This is an attempt to convey to the reader quickly that the problem is more widespread than they may have thought.

Reply:
```suggestion
When you develop with .NET, follow these recommendations when you compare strings

> [!TIP]
> Comparison includes string related methods, such as <xref:System.String.Equals%2A?displayProperty=nameWithType>, <xref:System.String.Compare%2A?displayProperty=nameWithType>, <xref:System.String.IndexOf%2A?displayProperty=nameWithType>, and <xref:System.String.StartsWith%2A?displayProperty=nameWithType>.
```

Reply:
I tweaked this suggestion slightly and ended up with https://github.com/dotnet/docs/pull/35069/commits/6fef7d5f8251c5e4e60bf956cc3a0ae35e217789 .